### PR TITLE
Add `assert_in_code` pattern

### LINKED ISSUE #119: assert_in_code pattern
Here it is:

```java
class Book {
  void foo(String x) {
    assert x != null; // here
  } 
}
```

Once you see the `assert` statement and class name doesn't end with `Test`, it's a pattern.

### PR DESCRIPTION
**Pull request by tribals**
Closes #119

### CODE REVIEW

**Comment by tribals** [comment]
@acheshkov Please, review.

**Comment by 0crat** [comment]
This pull request [#132](https://github.com/yegor256/aibolit/issues/132) is assigned to @fanifieiev[/z](https://www.0crat.com/u/fanifieiev), here is [why](https://www.0crat.com/footprint/CT5UB7XAP/09754c59-fb7d-4600-819f-a279a032db12); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @acheshkov[/z](https://www.0crat.com/u/acheshkov) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: aibolit/patterns/assert_in_code/assert_in_code.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+_TEST_CLASS_SUFFIX = 'Test'
+
+
+class AssertInCode(object):
+    def value(self, filename: str):
+        tree = self.__file_to_ast(filename)
+
+        return self.__traverse_node(tree)
+
+    def __file_to_ast(self, filename: str) -> javalang.ast.Node:
```
Comment (fanifieiev) [inline aibolit/patterns/assert_in_code/assert_in_code.py]: @tribals Just for you to know, this method is repeated in many places, so I consider it as a copy-paste, that's why I created a ticket https://github.com/yegor256/aibolit/issues/131 with a suggestion to improve that case.

Comment (tribals) [reply aibolit/patterns/assert_in_code/assert_in_code.py]: I agree. We need to invent some kind of interface/base class for patterns. Do you suppose that refactoring of already existing patterns should be done as part of this issue?

**Comment by acheshkov** [comment]
@fanifieiev Can I merge it?

File: aibolit/patterns/assert_in_code/assert_in_code.py (hunk 1 above)
Comment (fanifieiev) [reply aibolit/patterns/assert_in_code/assert_in_code.py]: @tribals No, this is just for you to know.

**Comment by acheshkov** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job audit: performer @fanifieiev[/z](https://www.0crat.com/u/fanifieiev)[/z](https://www.0crat.com/u/fanifieiev) didn't make CR comments

**Pull request merged**